index.html

```diff
@@ -218,6 +218,7 @@ <h4 class="proj_name">Project name goes here</h4>
               </div>
             </div>
           </div>
+          <!-- <div class="modal-overlay"></div> -->
```

Comment:
- [ ] As mentioned in the above comments, KIndly make sure you remove all the commented and unused code lines from your code base for **[best practice reasons](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**. This makes your project look messy.